compiler/one-cmds/one-codegen

```diff
@@ -109,8 +142,13 @@ def main():
     _verify_arg(parser, args)
 
     # make a command to run given backend driver
-    dir_path = os.path.dirname(os.path.realpath(__file__))
-    codegen_path = os.path.join(dir_path, getattr(args, 'backend') + '-compile')
+    codegen_path = None
+    backend_base = getattr(args, 'backend') + '-compile'
+    for cand in backends_list:
+        if ntpath.basename(cand) == backend_base:
+            codegen_path = cand
+    if not codegen_path:
+        raise FileNotFoundError(backend_base + ' not found')
```

Comment:
If this code does not excute with one command name, for bachends in `PATH`,
can you please add a `TODO` for that?